tests/iotools/test_pvgis.py

```diff
@@ -630,23 +622,6 @@ def test_read_pvgis_tmy_csv(expected, month_year_expected, inputs_expected,
                                meta_expected, csv_meta, pvgis_data)
 
 
-def test_read_pvgis_tmy_basic(expected, meta_expected):
-    fn = TESTS_DATA_DIR / 'tmy_45.000_8.000_2005_2023.txt'
```

Comment:
Can we ditch this `tmy_45.000_8.000_2005_2023.txt` file now?

Reply:
Yes, good catch!